docs/gatsby-node.js

```diff
@@ -24,4 +24,589 @@ exports.createPages = ({ actions }) => {
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-big-can-my-data-source-be',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-do-i-create-my-own-visualization',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#can-i-upload-and-visualize-csv-data',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#why-are-my-queries-timing-out',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#why-is-the-map-not-visible-in-the-mapbox-visualization',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-to-add-dynamic-filters-to-a-dashboard',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-to-limit-the-timed-refresh-on-a-dashboard',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#why-does-flask-fab-or-superset-freezed-hung-not-responding-when-started-my-home-directory-is-nfs-mounted',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#what-if-the-table-schema-changed',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-do-i-go-about-developing-a-new-visualization-type',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#what-database-engine-can-i-use-as-a-backend-for-superset',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-can-i-configure-oauth-authentication-and-authorization',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#how-can-i-set-a-default-filter-on-my-dashboard',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: 'faq.html#how-do-i-get-superset-to-refresh-the-schema-of-my-table',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#is-there-a-way-to-force-the-use-specific-colors',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/faq.html#does-superset-work-with-insert-database-engine-here',
+    toPath: '/docs/frequently-asked-questions',
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: '/index.html',
+    toPath: '/docs/intro',
+    isPermanent: true,
+  });
+};
```

Comment:
Looks like you need a newline at the end of the file here... `npm run lint-fix` from the `/docs` folder _should_ do it for ya, but this might be a good excuse to check.